### PR TITLE
[Issue 12] Spam Handling

### PR DESCRIPTION
**Pull request by mauricerkelly**
In response to Issue #12 and in light of events this last week, I've written up some stuff about what we consider spam and how we will deal with it.

### CODE REVIEW

**Review by MarkXA — APPROVED** [review]

**Review by clairebones — APPROVED** [review]

**Comment by mauricerkelly** [comment]
@andrewbolster I'd feel happier about merging this with your approval, since you raised the issue that it was based on.

File: code-of-conduct.md [hunk 1]
```diff
@@ -62,6 +62,19 @@ Participants asked to stop any harassing behaviour are expected to comply immedi
...
+----
+
+Within the context of this Slack spam is defined as behaviour that involves the posting of unsolicited adverts, or the posting of excessive amounts of material to any channel or feature of the Slack. Examples of spam include (but are not limited to):
+
+- repeated advertising of services or products
+- excessive uploads of customisable material (for example: emoji, Slackbot responses, loading messages)
+- joining the service solely to advertise
+- running bots that post excessively or cause annoyance to other members
+
+In the event of an activity considered to be spam being reported to the admins, a request will be made to the member responsible to rectify their actions (by removing messages, emoji, bots, etc) and to not repeat the activity. Failure to rectify the actions will result in the admins carrying this out. Repeating the activity will result in account deactivation.
```
Comment (andrewbolster) [inline code-of-conduct.md]: could do with an addition of 'within a timely fashion' to the "Failure to rectify" phrase.  Other than that LGTM

**Review by andrewbolster — CHANGES_REQUESTED** [review]

**Comment by mauricerkelly** [comment]
@andrewbolster requested change has been carried out.

**Comment by mauricerkelly** [comment]
@andrewbolster still waiting on a re-review from you. Cheers.

**Review by andrewbolster — APPROVED** [review]

**Pull request merged**